src/firefly/robot.py

```diff
@@ -0,0 +1,170 @@
+import logging
+
+from bluesky_queueserver_api import BPlan
+from qtpy import QtWidgets
+
+from firefly import display
+from firefly.component_selector import ComponentSelector
+
+log = logging.getLogger(__name__)
+
+# ROBOT_NAMES = ["Austin"]
+
+SAMPLE_NUMBERS = [8, 9, 10, 14, 15, 16, 20, 21, 22, None]  # list(range(24))+[None]
```

Comment:
We shouldn't hard-code this list of which samples are enabled in the UI layer. Can we get this information from the ``Robot()`` device somehow?